client/document/index.jsx

```diff
@@ -82,7 +83,10 @@ class Document extends Component {
 			( languageRevisions
 				? `var languageRevisions = ${ jsonStringifyForHtml( languageRevisions ) };\n`
 				: '' ) +
-			`var installedChunks = ${ jsonStringifyForHtml( installedChunks ) };\n`;
+			`var installedChunks = ${ jsonStringifyForHtml( installedChunks ) };\n` +
+			( params && params.hasOwnProperty( 'lang' )
```

Comment:
Could we add a comment that connects this to where the `lang` is [extrapolated](https://github.com/Automattic/wp-calypso/blob/trunk/packages/i18n-utils/src/utils.ts#L119)?

[nit] I think `localeFromRoute` is more descriptive here since we can associate it with route parameters, WDTY?

Reply:
Renamed!